bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -8,14 +8,14 @@ metadata:
     certified: "false"
     createdAt: REPLACE_DATE
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=1.0.0 <2.0.0'
+    olm.skipRange: '>=2.0.0 <3.0.0'
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
-    operators.operatorframework.io/builder: operator-sdk-v2.0.0+git
```

Comment:
@sebsoto why was this ever `v2.0.0`?

Reply:
Thats what Is generated. @mansikulkarni96 was this change generated or done manually?

Reply:
this is auto-generated on `make bundle`, even if I try changing it to 2.0.0, it is stomped back to 1.6.1 that points to the SDK version.

Reply:
Thats not the behavior I see:

```
[sebsoto@fedora windows-machine-config-operator]$ make bundle WMCO_VERSION=3.0.0
/home/sebsoto/code/openshift/windows-machine-config-operator/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false" rbac:roleName=windows-machine-config-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate kustomize manifests -q
WARN[0000] Skipping definitions parsing: APIs root dir "api" does not exist 
cd config/manager && /home/sebsoto/code/openshift/windows-machine-config-operator/bin/kustomize edit set image controller=REPLACE_IMAGE
/home/sebsoto/code/openshift/windows-machine-config-operator/bin/kustomize build config/manifests | operator-sdk generate bundle --overwrite=false -q --version 3.0.0 --channels="preview,stable" --default-channel="stable"
operator-sdk bundle validate ./bundle
INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
INFO[0000] All validation tests have completed successfully 
sed -i 's/windows-machine-config-operator\.v.\.0\.0/windows-machine-config-operator.v3.0.0/g' ./bundle/windows-machine-config-operator.package.yaml
[sebsoto@fedora windows-machine-config-operator]$ git diff
diff --git a/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml b/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
index 77143a85..f82d8c8c 100644
--- a/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
@@ -15,7 +15,7 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/windows-machine-config-operator
     support: Red Hat
-  name: windows-machine-config-operator.v2.0.0
+  name: windows-machine-config-operator.v3.0.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
@@ -302,7 +302,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: windows-machine-config-operator
-                image: REPLACE_IMAGE
+                image: REPLACE_IMAGE:latest
                 imagePullPolicy: IfNotPresent
                 name: manager
                 resources: {}
@@ -366,4 +366,4 @@ spec:
   minKubeVersion: 1.21.0
   provider:
     name: Red Hat
-  version: 2.0.0
+  version: 3.0.0
diff --git a/bundle/windows-machine-config-operator.package.yaml b/bundle/windows-machine-config-operator.package.yaml
index b8a9164c..b02e3eae 100644
--- a/bundle/windows-machine-config-operator.package.yaml
+++ b/bundle/windows-machine-config-operator.package.yaml
@@ -1,4 +1,4 @@
 channels:
-  - currentCSV: windows-machine-config-operator.v2.0.0
+  - currentCSV: windows-machine-config-operator.v3.0.0
     channels: alpha
 packageName: windows-machine-config-operator
```

Reply:
Not the case for me, maybe someone else should verify. I am unsure although why the builder would point to the WMCO version, also I looked at some of the operators using the bundle format and they point to the sdk vesrion as well.

Reply:
@sebsoto your comment is talking about the operator version while my comment was about the SDK version. Let's get this resolved during parking lot.

Reply:
I am talking about the SDK version as well, but yes we should look into this today at standup

Reply:
I am seeing the same behavior as @sebsoto:
```bash
windows-machine-config-operator   master → origin ↑2 make bundle WMCO_VERSION=3.0.0
/home/aravindh/devel/github/openshift/windows-machine-config-operator/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false" rbac:roleName=windows-machine-config-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate kustomize manifests -q
WARN[0000] Skipping definitions parsing: APIs root dir "api" does not exist 
cd config/manager && /home/aravindh/devel/github/openshift/windows-machine-config-operator/bin/kustomize edit set image controller=REPLACE_IMAGE
/home/aravindh/devel/github/openshift/windows-machine-config-operator/bin/kustomize build config/manifests | operator-sdk generate bundle --overwrite=false -q --version 3.0.0 --channels="preview,stable" --default-channel="stable"
operator-sdk bundle validate ./bundle
INFO[0000] Found annotations file                        bundle-dir=bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=bundle container-tool=docker
INFO[0000] All validation tests have completed successfully 
sed -i 's/windows-machine-config-operator\.v.\.0\.0/windows-machine-config-operator.v3.0.0/g' ./bundle/windows-machine-config-operator.package.yaml
 windows-machine-config-operator   master → origin ↑2 U:2  git diff
diff --git a/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml b/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
index d097741..7886759 100644
--- a/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml
@@ -15,7 +15,7 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/windows-machine-config-operator
     support: Red Hat
-  name: windows-machine-config-operator.v2.0.0
+  name: windows-machine-config-operator.v3.0.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
@@ -276,7 +276,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: windows-machine-config-operator
-                image: REPLACE_IMAGE
+                image: REPLACE_IMAGE:latest
                 imagePullPolicy: IfNotPresent
                 name: manager
                 resources: {}
@@ -340,4 +340,4 @@ spec:
   minKubeVersion: 1.21.0
   provider:
     name: Red Hat
-  version: 2.0.0
+  version: 3.0.0
diff --git a/bundle/windows-machine-config-operator.package.yaml b/bundle/windows-machine-config-operator.package.yaml
index b8a9164..b02e3ea 100644
--- a/bundle/windows-machine-config-operator.package.yaml
+++ b/bundle/windows-machine-config-operator.package.yaml
@@ -1,4 +1,4 @@
 channels:
-  - currentCSV: windows-machine-config-operator.v2.0.0
+  - currentCSV: windows-machine-config-operator.v3.0.0
     channels: alpha
 packageName: windows-machine-config-operator
```

Reply:
I am seeing the same behavior as both of you now that I have re-installed my operator-sdk, I will update the PR.